blocks/inspector-controls/textarea-control/index.js

```diff
@@ -21,10 +21,9 @@ function TextareaControl( { label, value, help, instanceId, onChange, rows = 4,
 				rows={ rows }
 				onChange={ onChangeValue }
 				aria-describedby={ !! help ? id + '__help' : undefined }
+				value={ value }
 				{ ...props }
-			>
```

Comment:
I guess the closing `textarea` tag is not needed any more

Reply:
@youknowriad Thanks for your review. I removed the closing tag.